.github/workflows/build.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
 
         # Intentionally use specific versions instead of "latest" to
         # make this build reproducible.
-        os: ['ubuntu-22.04', 'macos-12']
+        os: ['ubuntu-22.04', 'macos-12', 'windows-2022']
```

Comment:
Can't we just specify `GOOS` before running the tests in order to run on a different platform, instead of using different runner types?

Reply:
Synced offline. With running on a different platform would require emulation and can be evaluated seperately.